app/database/queries/breeze_buddy/users.py

```
@@ -51,10 +117,617 @@ async def get_user_by_username(username: str) -> Optional[UserInDB]:
+            if row:
+                logger.info(f"Updated user: {user_id}")
+                return _build_user_response(row)
+
+            return None
+
+    except Exception as e:
+        logger.error(f"Error updating user {user_id}: {e}")
+        raise
+
+
+async def delete_user(user_id: str, caller_role: UserRole) -> bool:
```

Ruff (0.15.0)

[warning] Unused function argument: `caller_role`

(ARG001)